scaleway/resource_instance_server.go

```diff
@@ -49,7 +49,7 @@ func resourceScalewayInstanceServer() *schema.Resource {
 				ForceNew:         true,
 				Description:      "The UUID or the label of the base image used by the server",
 				DiffSuppressFunc: diffSuppressFuncLocality,
-				AtLeastOneOf:     []string{"image", "root_volume.0.volume_id"},
+				ExactlyOneOf:     []string{"image", "root_volume.0.volume_id"},
```

Comment:
Can you add this precision on the doc.  May a small section about this could be better. The instance's doc is already big

Reply:
This is already documented

Reply:
https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/instance_server#image

Reply:
https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/instance_server#image